documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -991,10 +991,10 @@ CacheEntry is returned, this entries value is "put" to all replicas in the cache
 returns a null value, all replicas associated with the conflicting key are removed from the cache.
 
 The merge method takes two parameters: the "preferredEntry" and "otherEntries". In the context of a partition merge,
-the preferredEntry is the CacheEntry associated with the partition whose coordinator is conducting the merge (or if
-multiple entries exist in this partition, it's the primary replica). However, in all other contexts, the preferredEntry
-is simply the primary replica. The second parameter, "otherEntries" is simply a list of all other entries associated
-with the key for which a conflict was detected.
+the preferredEntry is the primary replica of a CacheEntry stored in the partition that contains the most nodes or if
+partitions are equal, the one with the largest topologyId. When a partition merge is not occurring, the "preferredEntry"
```

Comment:
I changed the algorithm slightly with ISPN-8962: we also pick the partition if the largest topology id if the partitions are overlapping (e.g. when partition [ABC](tid=1) merges with partition [A](tid=3), the preferred partition is [A]).